src/hotspot/share/runtime/thread.cpp

```diff
@@ -274,8 +274,6 @@ Thread::Thread() {
   _visited_for_critical_count = false;
 #endif
 
-  _SR_lock = new Monitor(Mutex::suspend_resume, "SR_lock", true,
```

Comment:
Mutex::suspend_resume rank is not used by any other monitor so we could remove it.

Reply:
Fixed